src/generic/hooks/index.jsx

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
If you find yourself writing this ^ (and I see it twice here in your PR so far), please consider supporting my PR https://github.com/openedx/eslint-config/pull/164 :)

Reply:
I completely agree, I also thought that we have a huge number of exceptions that we could remove. Thank you!